requirements/base.txt

```diff
@@ -12,6 +12,7 @@ django-bootstrap-form==3.2
 croniter==0.3.12
 GitPython==1.0.2
 gevent-socketio==0.3.6
+gevent==1.2.1
```

Comment:
I think this is invalid, `gevent-socketio` has `gevent` as an dependency. This could possible lead to incompatibilities if we upgrade gevent-websocket to a newer version without bumping gevent.